packages/wxt/src/core/utils/building/find-entrypoints.ts

```diff
@@ -151,6 +154,12 @@ export async function findEntrypoints(): Promise<Entrypoint[]> {
         .join('\n')}`,
     );
   }
+
+  if (allowAllEntryPoints) {
+    wxt.config.zip.excludeSources = entrypoints.map((entry: Entrypoint) => {
+      return entry.inputPath;
+    });
+  }
```

Comment:
Here, two changes:

1. You should only be adding excluded entrypoints to the `excludeSources`, but you're adding all entrypoints. See the filter right below this.
2. Instead of overwriting the `excludeSources` entirely, you need to merge the existing array with the skipped entrypoints. Note that this function is called multiple times, so you should use a `Set` to do this, not `Array.push`, which is why I used the term "merge".